RevenueCatUI/PaywallView.swift

```diff
@@ -305,14 +302,19 @@ struct LoadedOfferingPaywallView: View {
             .disabled(self.purchaseHandler.actionInProgress)
             .onAppear { self.purchaseHandler.trackPaywallImpression(self.createEventData()) }
             .onDisappear { self.purchaseHandler.trackPaywallClose() }
+            .onChangeOf(self.purchaseHandler.purchased) { purchased in
+                if purchased {
+                    Logger.debug(Strings.dismissing_paywall)
+                    self.dismiss()
```

Comment:
I wonder if this could be considered a breaking change somewhat... Since people might have had to handle this before, would they be dismissing the paywall twice with this now?

Reply:
Even if they do, I tested it and there is no difference :) The double-dismissal becomes a no-op.